content/html/concepts/attributes/terms/dir/dir.md

```diff
@@ -0,0 +1,51 @@
+---
+Title:
+- 'dir'
+Description:
+- 'Specifies the text direction of the content of an element.'
+Subjects:
+- 'Web Development'
+- 'Web Design'
+Tags:
+- 'Attribute'
+- 'Value'
+Catalog Content:
+- 'learn-html'
+- 'paths/front-end-engineer-career-path'
+---
+
+The **`dir`** global attribute specifies the text direction of an HTML element's content.
+
+The `dir` attribute is used to set the base direction of text for display. It is essential to support languages that use right-to-left scripts. Many different languages are written with these scripts, including Adlam, Arabic, Dhivehi, Hebrew, Mandinka, N'Ko, Pashto, Persian, Pular, Sindhi, Syriac, Thaana, Urdu, Yiddish, etc.  
+
+## Syntax:
+
+```html
+<element  dir="ltr | rtl | auto">
+```
+
+## Possible Attribute Values: 
```

Comment:
Can we remove this header and replace it with a sentence like, "These are the possible attribute values:"

Reply:
@SSwiniarski I'm sorry I misunderstood which way you wanted it written.
Just to clarify, I only need to make one initial PR to any remote repo, and subsequently only pushes will be necessary , once I pull down any updates?
Did I finally get the project completed?

Reply:
Yes you only need the one PR for this entry. If you decide to add _another_ entry, then you create a new branch of your main fork and do another PR.